elastic/integrations
# [Cloud Security] Fix ECS import method

## What does this PR do?


This PR fixes the way `cloud_security_posture` imports ECS mappings. Till now, we did it manually without knowing how to instruct `elastic-package` which ECS version to import its mappings from.

## Checklist


## Author's Checklist



## How to test this PR locally


1. Build locally the package
1. Install version 1.2.10-next
2. Check mappings are as before

Tested with **elastic-package 0.72** (requires **go 1.19** when built locally)

## Related issues


-

## Screenshots
